docs/ops/doc/WhyOps.md

```diff
@@ -0,0 +1,46 @@
+
+## What are the driving values of SciJava Ops?
+
+1. **Ease of use**:  All Ops are called in the same way, regardless of the mechanisms used by the underlying framework. This means that you don't have to learn Python to call Ops written in Python, but it also means that you could pass the output from an Op written in Python to an Op written in Java, all with the same syntax!
+2. **Reusability**: Ops extends Java's mantra of "write once, run anywhere" to image processing algorithms. Algorithms written for the SciJava Ops framework are usable as-is from any SciJava-compatible software project including ImageJ2/Fiji or from Python using PyImageJ. 
+3. **Reproducibility**: Ops are deterministic: calling the same op twice with the same arguments yields the same result, always. Ops are also versioned, meaning that if you use the same Op environment with the same library versions, you will always have reproducible workflows.
+4. **Power**: An op may consist of any number of strongly typed inputs. Ops may operate on arbitrary data structures, including images of N dimensions stored in a myriad of different ways.
+5. **Extensibility**: Ops provides a framework for writing new ops with zero additional code, and for extending existing ops in new directions.
+6. **Speed**: The Ops framework provides a means to override any general-but-slow op with a faster-but-more-specific alternative and the execution framework adds minimal overhead.
+
+## How do we integrate all of these different libraries and data structures?
+
+Core to SciJava Ops is the observation that algorithmic data structures generally fall into a one of a set of "forms", including:
+* scalars
+* sets
+* tensors
+* images
+* ...
+
+To enable the goals described above, SciJava Ops provides an interface for seamless conversion between implementations of different forms, which can be optimized for performance and/or efficiency. This allows us to write scripts calling algorithms in different languages and libraries.
+
+To enable a uniform interface for algorithm execution, we extend the notion of "form"s to algorithms as well, as human intuition is good at determining whether an algorithm is a "gaussian blur", a "convolution", a "segmentation", and so on.
+
+Within each algorithm "form", there are several different sub-algorithms - for example, for a gaussian blur, you might see algorithms requiring
+* an image and a single sigma
+* an image and a sigma per dimension
+* an image and a shape over which to compute the gaussian blur
+* ...
+
+It is SciJava Ops' goal to collapse all algorithms into the following descriptions, and to provide a uniform execution pathway for each - hiding the complexities of each individual algorithm library.
+
+## Isn't SciJava Ops just another algorithm library?
+
+While it may currently seem that way, SciJava Ops is designed to *wrap* existing algorithms, not to create new ones (although you can certainly do that!)
+
+To that end, SciJava Ops 
```

Comment:
@gselzer did you have more written that goes here? Do you have more sections planned?

Reply:
@hinerm I just wanted to write something along the lines of "SciJava exposes a rich API for declaring Ops from arbitrary sources, spanning libraries and programming languages". Sorry I forgot!